bokehjs/src/lib/models/widgets/abstract_button.ts

```diff
@@ -59,7 +59,12 @@ export abstract class AbstractButtonView extends ControlView {
     this.button_el.addEventListener("click", () => this.click())
 
     if (this.icon_view != null) {
-      prepend(this.button_el, this.icon_view.el, nbsp())
+      if (this.model.label!==""){
+        prepend(this.button_el, this.icon_view.el, nbsp())
+      }
+      else {
+        prepend(this.button_el, this.icon_view.el)
+      }
```

Comment:
This could be achieved more cleanly (avoid code duplication between branches) with:
```ts
prepend(this.button_el, this.icon_view.el, this.model.label != "" ? nbsp() : null)
```

Reply:
Actually, scratch that. That only works with element constructor functions unfortunately.

Reply:
In any case, please follow bokehjs' code style, i.e. space around operators and before `{`, `else` branch on the same line as the closing `}`. Also, `label` is non-nullable, so it's sufficient to use `!=` operator.